builder/build.go

```diff
@@ -1428,6 +1434,23 @@ func printStacks(calculatedStacks []string, stackSizes map[string]functionStackS
 	}
 }
 
+func applyPatches(executable string, bootPatches []string) (err error) {
+	for _, patch := range bootPatches {
+		switch patch {
+		case "rp2040":
+			err = patchRP2040BootCRC(executable)
+		// case "rp2350":
```

Comment:
Is this supposed to be commented out here?

Reply:
No patch required for rp2350 from my understanding

Reply:
Then we should remove this, no?